src/Components/HistoricalDataChart/HistoricalDataChart.scss

```diff
@@ -0,0 +1,36 @@
+.chartContainer{
+   display: flex;
+   flex-direction: column;
+   align-items: flex-start;
+   position: absolute;
+   top: 0;
+   right: 0;
+   margin-right: 8%;
+}
+
+.dropdownContainer {
+   margin-right: 200px;
+   align-self: flex-end;
+}
+
+.dateDropdown{
+   padding-left: 8px;
+}
+
+.loading {
+   width: 756px;
+   height: 275px;
+}
+
+// Media Query
+@media (max-width: 1620px ){
```

Comment:
Can we make it to `1420px`? And hence remove `margin-right: 8%;` from line 8.

Reply:
Hello @r14chandra the reason I kept this value is because less than that the UI feels like bit overlapping for eg here is screenshot with width 1520 

![image](https://user-images.githubusercontent.com/5928530/165731854-d7e4680d-f0ba-40a8-af43-42a0171cbc4f.png)

and the reason I kept margin 8% is because Stefan mentioned that the legends should be bit far from Actions dropdown. I hope that answers your question.

Reply:
Got it! Thanks @PreetiW